internal/api/attestationconfig/client/client_test.go

```diff
@@ -1,82 +1,51 @@
-//go:build e2e
-
 /*
 Copyright (c) Edgeless Systems GmbH
 
 SPDX-License-Identifier: AGPL-3.0-only
 */
-package client_test
+package client
 
 import (
-	"context"
-	"flag"
-	"fmt"
-	"io"
-	"os"
+	"encoding/json"
 	"testing"
-	"time"
 
 	"github.com/edgelesssys/constellation/v2/internal/api/attestationconfig"
-	"github.com/edgelesssys/constellation/v2/internal/api/attestationconfig/client"
-	"github.com/edgelesssys/constellation/v2/internal/staticupload"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-const (
-	awsBucket   = "cdn-constellation-backend"
-	awsRegion   = "eu-central-1"
-	envAwsKeyID = "AWS_ACCESS_KEY_ID"
-	envAwsKey   = "AWS_ACCESS_KEY"
-)
-
-var cfg staticupload.Config
-
-var (
-	cosignPwd      = flag.String("cosign-pwd", "", "Password to decrypt the cosign private key. Required for signing.")
-	privateKeyPath = flag.String("private-key", "", "Path to the private key used for signing. Required for signing.")
-	privateKey     []byte
-)
-
-func TestMain(m *testing.M) {
-	flag.Parse()
-	if *cosignPwd == "" || *privateKeyPath == "" {
-		flag.Usage()
-		fmt.Println("Required flags not set: --cosign-pwd, --private-key. Skipping tests.")
-		os.Exit(1)
-	}
-	if _, present := os.LookupEnv(envAwsKey); !present {
-		fmt.Printf("%s not set. Skipping tests.\n", envAwsKey)
-		os.Exit(1)
+// TODO replace with unit testable version
```

Comment:
Fix TODO or add ticket reference and/or add person responsible to implement this

Reply:
i wanted to get feedback on this way of testing. You think we should use it in all parts?

Reply:
I prefer having unit tests over integration tests that need to be ran manually in packages.
That being said, it might still be helpful to have an integration test to test the functionality of a package against a real API.
If you want to keep a test like that, I suggest moving it into a separate directory and package, e.g. `internal/api/attestationconfig/client/test/integration_test.go

Reply:
I like the idea for testing, but would not extend it to the other tests within this PR.